embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/ToStringMap.java

```diff
@@ -32,4 +21,17 @@ public Properties toProperties()
         props.putAll(this);
         return props;
     }
+
+    private static Map<String, String> mapToStringString(final Map<String, ToString> mapOfToString) {
+        final HashMap<String, String> result = new HashMap<>();
+        for (final Map.Entry<String, ToString> entry : mapOfToString.entrySet()) {
+            final ToString value = entry.getValue();
+            if (value == null) {
+                result.put(entry.getKey(), null);
```

Comment:
The old one is `null` string, the new is null, is it side effect somewhere?

Reply:
Ah, good catch! Will fix it.

Reply:
Fixed at https://github.com/embulk/embulk-input-jdbc/pull/207/commits/bc8ebcb911f6dd3d817521b04142548f5fe5d228#diff-4682fe5aa166d0303b1cde0cac83369807c974f61ff18f6981be2aa0c4bbc895R29